### PR TITLE
Information

### PR DESCRIPTION
**Pull request by Seggan**
## Changes

Added information about planets, refactored the old `GalacticExplorer`, actually removed `int` chunk versions, added 2 more machines

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break world generation, mob spawning, and the like.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values

### CODE REVIEW

**Review by Mooy1 — COMMENTED** [review]
LGTM

**Review by Mooy1 — DISMISSED** [review]
forgot to click approve

**Comment by Seggan** [comment]
Oops didn't see the approval. Well then let's add this into the mix

File: src/main/java/io/github/addoncommunity/galactifun/api/events/PlayerVisitWorldEvent.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import lombok.Getter;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+
+import io.github.addoncommunity.galactifun.api.worlds.PlanetaryWorld;
+
+public class PlayerVisitWorldEvent extends PlayerEvent implements Cancellable {
```
Comment (Mooy1) [inline src/main/java/io/github/addoncommunity/galactifun/api/events/PlayerVisitWorldEvent.java:14]: not sure about adding this, isnt there already PlayerJoinWorldEvent? If they want to cancel joining it they can use that. For the discovery/knowledge stuff we should just call it internally

**Comment by Seggan** [comment]
Intellij PR feature being weird

File: src/main/java/io/github/addoncommunity/galactifun/core/managers/WorldManager.java [hunk 2]
```diff
@@ -325,4 +325,9 @@ public void onPlayerPlaceWater(PlayerBucketEmptyEvent e) {
         }
     }
 
+    @EventHandler(ignoreCancelled = true)
+    public void onPlayerJoinGame(PlayerJoinEvent e) {
```
Comment (Mooy1) [inline src/main/java/io/github/addoncommunity/galactifun/core/managers/WorldManager.java:329]: What if the server starts people on another planet? we should require the normal system for earth

**Review by Mooy1 — CHANGES_REQUESTED** [review]
other than that looks good

File: src/main/java/io/github/addoncommunity/galactifun/core/managers/WorldManager.java (hunk 2 above)
Comment (Seggan) [reply src/main/java/io/github/addoncommunity/galactifun/core/managers/WorldManager.java:329]: yeah

**Pull request merged**